handlers/util.go

```diff
@@ -0,0 +1,21 @@
+package handlers
+
+import (
+	"fmt"
+	"os"
+)
+
+var isDebug = false
+
+func init() {
+	if val, exists := os.LookupEnv("debug"); exists && (val == "1" || val == "true") {
+		isDebug = true
+	}
+}
+
+// debug prints out debug message
+func debug(format string, a ...interface{}) {
```

Comment:
I'd probably avoid this for now as we are likely to be changing logging library soon.